api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/TxPvlsDataset.java

```diff
@@ -63,6 +61,69 @@ public DataSetDefinition constructTxPvlsDatset() {
             mappings),
         "");
 
+    dsd.addColumn(
```

Comment:
What is the reason for adding these extra columns? From what I remember this change was only to change the period...

Reply:
@nthfloor ,I think some one added these two columns for test purpose and forgot to delete 
(

> dsd.addColumn("P", "Pregnant". , dsd.addColumn("B", "Breastfeeding")

Reply:
I will remove these columns

Reply:
@Guimino these columns are still in this dataset?

Reply:
@nthfloor  I removed them from the dataset

Reply:
really I still see them when I pull latest...

Reply:
@nthfloor , make sure you are pulling the latest, @ningosi  can you see that columns ?